cypress/support/commands.ts

```diff
@@ -4,14 +4,17 @@ import '@testing-library/cypress/add-commands';
 
 Cypress.Commands.add('start', (path: string, withLogin: boolean = false) => {
   cy.visit(path);
+  cy.intercept({
+    method: 'GET',
+    url: 'https://patient.qa.clin.ferlab.bio/patient/search?page=1&size=25&type=prescriptions',
```

Comment:
We should use the env variable `REACT_APP_PATIENT_SERVICE_URL` instead of `https://patient.qa.clin.ferlab.bio/patient`